### ISSUE TITLE
JettyServletServer.stopServer throws IllegalStateException when the server is not yet stopped

### ISSUE THREAD

**Issue by andref** [kumuluz/kumuluzee#112]
I'm doing integration testing and I needed to programatically start and stop the `EeApplication`. I'm trying to create something like Dropwizard's `DropwizardTestSupport` class. In a nutshell, I'm doing something like this:

```java
class SomeIntegrationTest {

  private EeApplication app;

  @Before
  public void startServer() {
    config = createCustomEeConfig();
    app = new EeApplication(config);
  }

  @After
  public void stopServer() {
    app.getServer().stopServer();
  }

  @Test
  public void test() {
    // hit the API, etc.
  }
}
```

The `stopServer()` method always throws an `IllegalStateException`. Examining `JettyServletServer`, the cause becomes apparent. The code guarding against multiple attempts to stop the server is the same as the one used for `startServer()`:

```java 
@Override
public void stopServer() {

    // ...

    if (server.isStarted() || server.isStarting())
        throw new IllegalStateException("Jetty is not running stopped");

    // ...
}
```

By simmetry, it should really be:

```java 
@Override
public void stopServer() {

    // ...

    if (server.isStopped() || server.isStopping())
        throw new IllegalStateException("Jetty is already stopped");

    // ...
}
```

I created PR #111 to fix the issue.

**Comment by urbim** [comment]
Thanks for the explanation, I have left a comment on the PR.

By the way: Have you seen [KumuluzEE Arquillian Container](https://github.com/kumuluz/kumuluzee-testing/blob/master/kumuluzee-arquillian-container/README.md)? By the looks of things you may be implementing something that has already been done.

**Comment by andref** [comment]
> By the way: Have you seen KumuluzEE Arquillian Container?

Yes, thanks for mentioning, but for e2e testing (which is what I'm doing in this case) I think it's simpler to just run the app itself. I might however be in for disappointment if it's not possible to stop and start the app multiple times inside the same JVM process. If that happens, I'll switch to Arquillian.

**Comment by jmezna** [comment]
Fixed in #111
